examples/integration/BUILD

```diff
@@ -15,16 +15,51 @@ load(
 
 exports_files(["README.md"])
 
-xcodeproj(
-    name = "xcodeproj",
-    associated_extra_files = ASSOCIATED_EXTRA_FILES,
-    build_mode = "xcode",
-    config = CONFIG,
-    extra_files = EXTRA_FILES,
-    project_name = "Integration",
-    scheme_autogeneration_mode = SCHEME_AUTOGENERATION_MODE,
-    schemes = get_xcode_schemes(),
-    tags = ["manual"],
-    top_level_targets = XCODEPROJ_TARGETS,
-    unfocused_targets = UNFOCUSED_TARGETS,
-)
+# Example of xcodeproj cache warming
+#
+# 1. Register a single macOS execution platform in WORKSPACE
+#    register_execution_platforms("@build_bazel_apple_support//platforms:macos_x86_64")
```

Comment:
This makes the path the same for both (`x86_64`), does it actually force anything to be non-fat `x86_64` (after accounting for `--@build_bazel_rules_swift//swift:universal_tools`)? Like `wrapped_clang` or any other host tools?

Reply:
This forces any non-fat host tools to be compiled as `x86_64`, so at least they still continue to run if you have Rosetta installed on your (remote or local) executor.

Since the purpose is to ensure a "macos_universal" platform, you can also use `register_execution_platforms("@build_bazel_apple_support//platforms:macos_arm64")` instead, or don't install Rosetta at all, and the build will error out if you have any non-fat tools.